.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+// Coloque su configuración en este archivo para sobrescribir la configuración predeterminada y de usuario.
```

Comment:
Good stuff @franmolmedo, then let's remove this config file from this PR, and open a new case to study this.